include/CppUTestExt/MockFailure.h

```diff
@@ -48,6 +48,7 @@ class MockFailureReporter
     virtual UtestShell* getTestToFail();
 
     virtual void crashOnFailure() {crashOnFailure_ = true; }
+    virtual void clear() {crashOnFailure_ = false; }
```

Comment:
That doesn't work. I will drop the second test case and not need to use TEST_ORDERED, since this behavior is not intended.

However, I am a bit worried about possible side-effects...